scripts/ci/run-ci-tests.sh

```diff
@@ -259,6 +259,7 @@ if [ -n "$TRAVIS" ] || [ -n "$CIRCLECI" ]; then
        # GitHub Actions (and Cirrus CI) does not provide a real TTY and CRIU will fail with:
        # Error (criu/tty.c:1014): tty: Don't have tty to inherit session from, aborting
        make -C test/others/shell-job/ run
+       make -C test/others/criu-ns/ run
```

Comment:
Could you move this line outside the "if" condition? (these tests also work in the GitHub CI)

Reply:
done.